site/en/models.md

```diff
@@ -0,0 +1,105 @@
+saving to the right location or by running `make install-models-tesseract` when
+using `ocrd_all`:
+
+  * [equ](https://github.com/tesseract-ocr/tessdata_fast/raw/master/equ.traineddata)
+  * [osd](https://github.com/tesseract-ocr/tessdata_fast/raw/master/osd.traineddata)
+  * [eng](https://github.com/tesseract-ocr/tessdata_fast/raw/master/eng.traineddata)
+  * [deu](https://github.com/tesseract-ocr/tessdata_fast/raw/master/deu.traineddata)
+  * [frk](https://github.com/tesseract-ocr/tessdata_fast/raw/master/frk.traineddata)
+  * [script/Latin](https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Fraktur.traineddata)
+  * [script/Fraktur](https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Fraktur.traineddata)
+  * [@stweil's GT4HistOCR model](https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/fast/Fraktur_50000000.334_450937.traineddata)
+
+OCR-D's Tesseract wrapper,
+[ocrd_tesserocr](https://github.com/OCR-D/ocrd_tesserocr) and more
+specifically, the `ocrd-tesserocr-recognize` processor, expects the name of the
+model(s) to be provided as the `model` parameter. Multiple models can be
+combined by concatenating with `+`:
+
+```sh
+# Use the deu and frk models
+ocrd-tesserocr-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-TESS -p '{"model": "deu+frk"}'
+# Use the script/Fraktur model
+ocrd-tesserocr-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-TESS -p '{"model": "script/Fraktur"}'
+```
+
+## Ocropy / ocrd_cis
+
+An Ocropy model is simply the neural network serialized as with Python's pickle
+mechanism and are generally distributed in a gzipped form, with an `.pyrnn.gz`
+extension.
+
+Ocropy has a rather convoluted algorithm to look up models, so we recommend you
+explicitly set the `OCROPUS_DATA` variable to point to the directory with
+ocropy's models. E.g. if you intend to store your models in `$HOME/ocropus-models`, add the following
+to you `$HOME/.bashrc`: `export OCROPUS_DATA=$HOME/ocropus-models`.
+
+We recommend you download the following models, either by downloading and
+saving to the right location or by running `make install-models-ocropus` when
+using `ocrd_all`:
+
+  * [en-default.pyrnn.gz](https://github.com/zuphilip/ocropy-models/raw/master/en-default.pyrnn.gz)
+  * [fraktur.pyrnn.gz](https://github.com/zuphilip/ocropy-models/raw/master/fraktur.pyrnn.gz)
+  * [@jze's fraktur.pyrnn.gz](https://github.com/jze/ocropus-model_fraktur/raw/master/fraktur.pyrnn.gz) (save as `fraktur-jze.pyrnn.gz`)
+  * [@chreul's  LatinHist.pyrnn.gz](https://github.com/chreul/OCR_Testdata_EarlyPrintedBooks/raw/master/LatinHist-98000.pyrnn.gz)
+
+
+To use a specific model with OCR-D's ocropus wrapper in [ocrd_cis](https://github.com/cisocrgroup/ocrd_cis) and more specifically, the `ocrd-cis-ocropy-recognize` processor, use the `model` parameter:
+
+```sh
+ocrd-cis-ocropy-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-OCRO -p '{"model": "fraktur-jze.pyrnn.gz"}'
+```
+
```

Comment:
ocropy does not allow to combine several models (as opposed to Tesseract)?

Reply:
No.

Reply:
This desideratum is called _voting_ and has not been implemented in either pyrnn or clstm backends (and thus, both Ocropus and Kraken) to my knowledge. Only Tesseract and Calamari engines have it.